src/vmm/src/guest_config/cpuid/amd/indexing.rs

```diff
@@ -6,7 +6,9 @@ use std::mem::transmute;
 
```

Comment:
nit: I know it's optional now, but could be good annotating the commit message with a refactoring tag.

Reply:
It wasn't purely a refactor. Would it have been preferable to split out the commits?

Reply:
Now that you said that, I think so. It's always nice if a commit serves one single purpose.